storybook/pages/OnboardingLayoutPage.qml

```diff
@@ -103,10 +103,6 @@ SplitView {
                 logs.logEvent("OnboardingStore.mnemonicWasShown()")
             }
 
-            function removeMnemonic() { // -> void
```

Comment:
Why removed?

Reply:
The mnemonic is removed automagically by status-go already